plugins/out_es/es.c

```diff
@@ -62,8 +62,10 @@ static flb_sds_t add_aws_auth(struct flb_http_client *c,
     /* AWS Fluent Bit user agent */
     flb_http_add_header(c, "User-Agent", 10, "aws-fluent-bit-plugin", 21);
 
+    flb_plg_debug(ctx->ins, "Using AWS Service: %s", ctx->aws_service_name);
```

Comment:
nit: I don't know if there's any real value in having this debug log. If the auth fails, you get a response from AWS saying that the service name is incorrect IIRC.

Reply:
I think I deleted this line actually. It shouldn't be here. Let me check

Reply:
I think I forget to delete it. But yeah, it can be removed. Will remove it in next version